UPGRADE-3.0.md

```diff
@@ -468,6 +468,8 @@ UPGRADE FROM 2.x to 3.0
    interface.
    The `security.csrf.token_manager` should be used instead.
 
+ * The default value of the parameter `cookie_httponly` is now `true`.
```

Comment:
I don't know if the UPGRADE guide is the best place for this, but should we also add a brief phrase exaplining the consequences of this change?